src/main/java/com/watch/shop/app/model/entity/SmartWatch.java

```diff
@@ -0,0 +1,13 @@
+package com.watch.shop.app.model.entity;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.experimental.SuperBuilder;
+
+@Data
+@SuperBuilder(toBuilder = true)
+@EqualsAndHashCode(callSuper = true)
+public class SmartWatch extends Watch {
+    private boolean hasHeartRateMonitor;
+    private OS os;
```

Comment:
Os os it's acceptable.

But a little bit too short, let's try
private OperationSystem operationSystem